recipes/wasmtime-cpp/all/conanfile.py

```diff
@@ -47,23 +47,31 @@ def requirements(self):
     def package_id(self):
         self.info.clear()
 
+    @property
+    def _info(self):
+        return self if Version(conan_version).major < 2 else self.info
```

Comment:
Please remove, this is fixed in the next beta drop

Reply:
Fixed